src/main/java/com/iotiq/application/repository/ProductDemandRepository.java

```diff
@@ -26,6 +26,6 @@ public interface ProductDemandRepository extends JpaRepository<ProductDemand, UU
 
     Page<ProductDemand> findAllBySeller(Pageable pageable, Seller seller);
 
-    @Query("SELECT pd FROM ProductDemand pd left join pd.sustainability.skills skills  WHERE pd.id = :id AND pd.isActive = true AND (pd.seller IS NULL OR pd.seller = :seller) AND (pd.sustainability.skills is empty or skills in :skills)")
-    Optional<ProductDemand> findByIdAndIsActiveAndSellerIsNullOrSellerAndSustainabilitySkillsEmptyOrSustainabilitySkillsIn(UUID id, Seller seller, Set<Skill> skills);
+    @Query("SELECT pd FROM ProductDemand pd left join pd.sustainability.skills skills  WHERE pd.id = :id AND (pd.seller IS NULL OR pd.seller = :seller) AND (pd.sustainability.skills is empty or skills in :skills)")
+    Optional<ProductDemand> findByIdAndSellerIsNullOrSellerAndSustainabilitySkillsEmptyOrSustainabilitySkillsIn(UUID id, Seller seller, Set<Skill> skills);
```

Comment:
We can simplify this method name since we don't use query method any more.  We can do the same for other such methods. This can be done in another PR